Source/ACE/Entity/Vendor.cs

```diff
@@ -205,14 +191,14 @@ public void BuyValidateTransaction(ObjectGuid vendorid, List<ItemProfile> items,
             // calculate price. (both unique and item profile)
             foreach (WorldObject wo in uqlist)
             {
-                goldcost = goldcost + (uint)Math.Ceiling(SellRate * (wo.Value ?? 0) * (wo.StackSize ?? 1) - 0.1);
+                goldcost = goldcost + (uint)Math.Ceiling((SellPrice ?? 1) * (wo.Value ?? 0) * (wo.StackSize ?? 1) - 0.1);
```

Comment:
Is 1 really the right default if SellPrice is undefined?

Perhaps we should print a warning if SellPrice is undefined, as I don't think there is a good default.